Makefile

```diff
@@ -39,8 +39,12 @@ sanity-test:
 	go test -v ./test/sanity
 build: setup
 	CGO_ENABLED=0 GOOS=linux go build -a -ldflags ${LDFLAGS} -o _output/secrets-store-csi ./cmd/secrets-store-csi-driver
+build-windows: setup
+	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/secrets-store-csi.exe ./cmd/secrets-store-csi-driver
 image: build
 	docker build --no-cache -t $(IMAGE_TAG) -f Dockerfile .
+image-windows: build-windows
+	docker build --no-cache -t $(IMAGE_TAG) -f windows.Dockerfile .
```

Comment:
nit: not sure what is the best naming convention for Dockerfile. here are few from other projects to consider:
https://github.com/moby/moby/blob/master/Dockerfile.windows
https://github.com/kubernetes/kubernetes/blob/master/test/images/busybox/Dockerfile_windows

WDYT?

Reply:
Having the `*.Dockerfile` file naming convention makes it possible for editors to recognize this as a docker file and provides syntax highlighting. That was one of the reasons for naming it this way.

Reply:
ok that makes sense!